src/ResourceManager/EventHub/Commands.EventHub/Microsoft.Azure.Commands.EventHub.format.ps1xml

```diff
@@ -58,9 +58,9 @@
       </ListControl>
     </View>  
     <View>
-      <Name>Microsoft.Azure.Commands.EventHub.Models.EventHubAttributes</Name>
+      <Name>Microsoft.Azure.Commands.EventHub.Models.PSEventHubAttributes</Name>
```

Comment:
I think all the other model names will need to be changed in this file as well.

Reply:
Resolved

Reply:
Did you miss pushing a commit? I still see that model names need to be changed.

Reply:
my bad, yes I did missed a commit with all the changes. I have pushed these changes again.